lib/github_backend.rb

```diff
@@ -7,15 +7,16 @@
 require_relative 'event_collection'
 
 class GithubBackend
-  attr_accessor :logger
+  attr_accessor :logger, :start_date, :repos
 
   def initialize(_args = {})
```

Comment:
Good point. Again, this was code I inherited and haven't gotten around to cleaning this part yet. Since I'm updating the README with your suggestion, I'll go ahead and make this change too.